healthcare/healthcare/doctype/therapy_session/therapy_session.py

```diff
@@ -58,6 +61,14 @@ def on_cancel(self):
 
 		self.update_sessions_count_in_therapy_plan(on_cancel=True)
 
+	def after_insert(self):
+		therapy_plan = frappe.get_doc('Therapy Plan', self.therapy_plan)
+		if therapy_plan.therapy_plan_template:
+			plan_template = frappe.get_doc('Therapy Plan Template', therapy_plan.therapy_plan_template)
+			template = plan_template.nursing_checklist_template
```

Comment:
For therapy sessions, checklists should be linked to Therapy Type instead of Plan. Also, avoid `get_doc`